docs/zh-cn/translator.md

```diff
@@ -51,7 +51,8 @@ True
 pyecharts 已经封装了底层相关逻辑，对使用者是透明的。因此你可以像之前一样的使用。将回函函数对象通过 `add` 方法赋值到 echarts 配置字典中，这里的回调函数需满足以下条件之一：
 
 - 使用 `def` 定义的命名函数
-- `lambda` 表达式 
```

Comment:
I can add the support in next PR.

Reply:
OK ,I will remove related code in this PR